WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -292,7 +292,9 @@ class AbstractPostListViewController: UIViewController, WPContentSyncHelperDeleg
 
         if tableViewHandler.resultsController.fetchedObjects?.count > 0 {
             hideNoResultsView()
-            presentNoNetworkAlert()
+            if ignoringNetworkAlerts {
```

Comment:
I think your logic or naming is inverted here – we should be presenting the alert if we're _not_ ignoring network alerts? Perhaps we should name this parameter `userInteraction` as well.

Reply:
Yes, the naming is definitely backwards, I will update. I did name it `userInteraction` at first, but I think I prefer something a bit less specific to how the alert gets initiated, and more closely related to why we do or do not want the alert to be presented, if that makes any sense. I am not a native English speaker though, so 🤷‍♂️.